src/core/shared/locale/locale.ts

```diff
@@ -3,6 +3,7 @@ import en from './languages/en'; // Importing the English Locale
 export const placeholder = `_%_`;
 
 let defaultLanguage = 'en';
+//let defaultLanguage = 'de'; // Uncomment this line to set the default language to German
```

Comment:
No need to comment this here, just add de without // to locales and people can just type in the already declared variable above ;D

Reply:
No i guess the have to choose one of them

Reply:
Looks better